internal/services/compute/orchestrated_virtual_machine_scale_set_resource.go

```diff
@@ -259,6 +259,8 @@ func resourceOrchestratedVirtualMachineScaleSet() *pluginsdk.Resource {
 				Sensitive:    true,
 				ValidateFunc: validation.StringIsBase64,
 			},
+
+			"priority_mix_policy": OrchestratedVirtualMachineScaleSetPriorityMixPolicySchema(),
```

Comment:
Do you think we can rename this property to just `priority_mix`? The naming convention in the provider tends to drop trailing descriptive words, such as `policy`, for the code blocks due to potential redundant wording with child field names. 

```suggestion
			"priority_mix": OrchestratedVirtualMachineScaleSetPriorityMixPolicySchema(),
```

Reply:
Another possible change to keep up with conventions is to remove "priority" from the subproperties:

"base_regular_priority_count" -> "base_regular_count"
"regular_priority_percentage_above_base" -> "regular_percentage_above_base"

Since the sub properties are under the scope of "priority mix". I'll push the change for now and we can revert if needed

Reply:
LGTM